fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -148,6 +148,32 @@ struct RelocateShard {
 	    moveReason(DataMovementReason::INVALID) {}
 };
 
+struct GetTeamRequest;
+namespace data_distribution {
+class EligibilityCounter {
+public:
+	enum Type {
+		NONE = 0, // don't care about eligibility
+		LOW_CPU = 1,
+		LOW_DISK_UTIL = 2
```

Comment:
IIUC, if anyone wants to add another `Type` here, it should be `LOW_*** = 4`? It's a little hard to understand for someone like me not familiar with bit set operation. It may help if leaving some comments here.